demo/ts-browser.js

```diff
@@ -30,32 +30,63 @@ const workerFile = window.URL.createObjectURL(
   )
 );
 
+const load = sourceUrl => {
+  const xhr = XMLHttpRequest
```

Comment:
Why not use native `fetch`? I haven't written an AJAX request like this in maybe 15 years hah.